eng/Version.Details.xml

```diff
@@ -1,25 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.WindowsDesktop.App.Runtime.win-x64" Version="3.1.4">
+    <Dependency Name="Microsoft.WindowsDesktop.App.Runtime.win-x64" Version="3.1.2">
```

Comment:
Shouldn't this be 3.1.3?

Reply:
We're reverting to the 3.1.2 runtime in this branch rather than 3.1.3

Reply:
But why? 3.1.3 already shipped publicly, right?

Reply:
we decide to revert to 3.1.2 to avoid further change in stage 0.

Reply:
For VS only release, we don’t want to flow runtime automatically. Or a preview of this build would ship before the actual patch Tuesday. We only want to flow to runtime once before the GA of the build.

Also, it is very costly to flow a runtime without automatic flow